src/fontstash.h

```diff
@@ -886,14 +886,41 @@ static int fons__allocFont(FONScontext* stash)
 	return FONS_INVALID;
 }
 
+static FILE* fons__fopen(const char* filename, const char* mode)
+{
+#ifdef _WIN32
+	int len = 0;
+	int fileLen = strlen(filename);
+	int modeLen = strlen(mode);
+	if(fileLen == 0)
+		return NULL;
+	if(modeLen == 0)
+		return NULL;
+	wchar_t wpath[MAX_PATH];
```

Comment:
All variable definitions should go at the top of the scope (forgot to add in last review) (see FILE* f below too).